plutus-tx-plugin/plutus-tx-plugin.cabal

```diff
@@ -121,10 +121,6 @@ test-suite plutus-tx-tests
   if flag(use-ghc-stub)
     buildable: False
 
-  -- test output changes after 9.2, bad for golden tests
-  if impl(ghc >=9.3)
-    buildable: False
```

Comment:
Is this a related change?

Reply:
No but this was committed by accident in https://github.com/input-output-hk/plutus/pull/5528 and I need to remove it to run the tests for 9.6.